libethcore/BasicAuthority.h

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
It's all indirect dependencies, probably because of the still unresolved EVM bits.